src/snackbar/_snackbar.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "../variables";
+
+.mdl-snackbar {
+  position: absolute;
```

Comment:
I did absolute to try and make sure it is always at the bottom. If you put it as a child of the content container for instance, fixed fixes it to the content viewport and not the body. Absolute provided the most consistency when testing compared to fixed.

Yes, if you follow the docs and have it as a child of the body it shouldn't matter. But I'd rather have a bit of buffer-room for errors in implementation.